release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3544,4 +3544,37 @@ For more information, see xref:../nodes/clusters/nodes-cluster-enabling-features
 [id="ocp-4-12-13-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-12-14"]
+=== RHBA-2023:1858 - {product-title} 4.12.14 bug fix update
+
+Issued: 2023-04-24
+
+{product-title} release 4.12.14 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:1858[RHBA-2023:1858] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:1857[RHBA-2023:1857] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.12.14 --pullspecs
+----
+
+[id="ocp-4-12-14-features"]
+==== Features
+
+[id="ocp-4-12-14-cloud-provider-openstack-1.25"]
+===== Cloud provider OpenStack is updated to 1.25
+
+With this release, Cloud Provider {rh-openstack-first} is updated to 1.25.5.
+
+[id="ocp-4-12-14-summary-of-openstack-changes"]
+====== Overview of Cloud Provider 1.25 updates
```

Comment:
I am not 100% about a level 6 heading here. I didn't see anything in the style guide against it, so it is up to you, but I almost think it would be clearer to the reader if  you  keep it with the above and just say 

> With this release, Cloud Provider {rh-openstack-first} is updated to 1.25.5. Updates include:
*bullet
*bullet

Reply:
Change made.